rlclientlib/serialization/fb_serializer.h

```diff
@@ -177,7 +177,7 @@ namespace reinforcement_learning { namespace logger {
     }
 
     static int serialize(outcome_event& evt, flatbuffers::FlatBufferBuilder& builder,
-                         flatbuffers::Offset<fb_event_t>& retval, api_status* status) {
+                         flatbuffers::Offset<fb_event_t>& retval, const char* app_id, api_status* status) {
```

Comment:
Move app_id to outcome_event. app_id doesn't fit the current design of having all event related data in the event object itself.

Reply:
My understanding that app_id is supposed to identify traffic before joining, so it sounds like metadata is the right place for it.

Reply:
We don't pass any other metadata this way.

Reply:
oh maybe I misunderstood - I thought you are talking about moving appId to Event/OutcomeEvent fb messages. If you are talking only about outcome_event class, I agree.

Reply:
yes, I was not clear enough. I meant outcome_event. Sorry for that.

Reply:
Synced offline with Alexey. I will make the changes Rodrigo suggested and update this PR.

Reply:
Moved app_id to generic_event.